src/xcode/ENA/ENA/Source/Views/BaseElements/ENAButton.swift

```diff
@@ -65,7 +65,10 @@ class ENAButton: DynamicTypeButton {
 		cornerRadius = 8
 
 		contentEdgeInsets = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 8)
-		heightAnchor.constraint(greaterThanOrEqualToConstant: 50).isActive = true
+		let heightConstraint = heightAnchor.constraint(greaterThanOrEqualToConstant: 50)
```

Comment:
Does somebody know why we need to have 50.0 height constraint for all ENAButtons ?

Reply:
@SebastianWolf-SAP

Reply:
I guess because it is defined in the designs guides that a button must have a height of 50pt or more. ;-)